pom.xml

```diff
@@ -1,9 +1,86 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.openmrs</groupId>
-  <artifactId>openmrs-content-hiv</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <name>Archetype - openmrs-content-hiv</name>
-  <url>http://maven.apache.org</url>
+	<modelVersion>4.0.0</modelVersion>
+ 	<groupId>org.openmrs.content</groupId>
+  	<artifactId>openmrs-content-hiv</artifactId>
+  	<version>1.0-SNAPSHOT</version>
+  	<name>OpenMRS HIV Content Package</name>		
+	<packaging>pom</packaging>
+	<developers>
+		<developer>
+			<name>OpenMRS</name>
+			<organization>OpenMRS</organization>
+			<organizationUrl>http://openmrs.org</organizationUrl>
+		</developer>
+	</developers>
+	<scm>
+		<connection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</connection>
+		<developerConnection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</developerConnection>
+		<url>https://github.com/openmrs/openmrs-content-hiv.git</url>
+		<tag>HEAD</tag>
+	</scm>	
+	<dependencies>
+		<dependency>
+            <groupId>org.openmrs.maven.plugins</groupId>
+            <artifactId>openmrs-packager-maven-plugin</artifactId>
+            <version>1.8.0-SNAPSHOT</version>
+        </dependency>
+	</dependencies>
+	<build>
+		<plugins>			
+			<plugin>
+				<groupId>org.openmrs.maven.plugins</groupId>
+				<artifactId>openmrs-packager-maven-plugin</artifactId>
+				<version>1.8.0-SNAPSHOT</version>
+				<executions>
+					<execution>
+						<id>validate-assemble</id>
+						<phase>package</phase>						
```

Comment:
```suggestion
						<phase>package</phase>
```

Reply:
I also think this should probably be run in the `verify` phase rather than the `package` phase. Technically, that comes after the Zip is generated, but in production usage, we're doing to be running the `deploy` phase to publish this, and that will block that.